multichain-testing/scripts/fetch-starship-chain-info.ts

```diff
@@ -50,7 +50,7 @@ const chainInfo = await convertChainInfo({
 });
 
 const record = JSON.stringify(chainInfo, null, 2);
-const src = `/** @file Generated by fetch-starship-chain-info.ts */\nexport default /** @type {const} } */ (${record});`;
+const src = `/** @file Generated by fetch-starship-chain-info.ts */\nexport default /** @type {Record<import("./test/support.js").ChainName, import("@agoric/orchestration").CosmosChainInfo>} } */ (${record});`;
```

Comment:
this drops information that is necessary for feature infererence. what motivated this change?

Reply:
https://github.com/Agoric/agoric-sdk/pull/9666#discussion_r1680129456